### DISCUSSION TITLE
Kubernetes Cluster API (CAPI) – An introduction

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#23] [BlogPosts]
# Kubernetes Cluster API (CAPI) – An introduction

The Kubernetes Cluster API aims to simplify the creation, configuration, upgrade, downgrade, and teardown of Kubernetes clusters and their components. It aims to do so across multiple providers, infrastructure types and enable uniformity, interoperability, extensibility, and simplicity..

https://pradeepl.com/blog/kubernetes/kubernetes-cluster-api-capi-an-introduction/

**Comment by Dhairya-Arora01** [comment]
Great intro !
